src/main/java/com/stripe/model/Card.java

```diff
@@ -221,6 +227,18 @@ public void setTokenizationMethod(String tokenizationMethod) {
 		this.tokenizationMethod = tokenizationMethod;
 	}
 
+	public String getDescription() { return description; }
```

Comment:
Committed a small patch in ad217ff to give these getter/setters a little more breathing room.